docs/cookbook/dump-variables.md

```diff
@@ -0,0 +1,16 @@
+# Dump variables in framework
```

Comment:
```suggestion
# Dump Variables
```

Reply:
We have some rules for writing docs, just like this one, see:
https://docs.shopsys.com/en/latest/contributing/guidelines-for-writing-documentation/

Maybe it would not be bad to read all contributing articles in our docs:
https://docs.shopsys.com/en/latest/contributing/